docs/stacks-guide-azure.md

```diff
@@ -0,0 +1,429 @@
+metadata:
+  name: standard-cluster
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: AKSClusterClass
+  apiVersion: compute.azure.crossplane.io/v1alpha2
+  name: azure-aks-standard
+  namespace: azure-infra-dev
+---
+EOF
+
+kubectl apply -f namespace.yaml
+```
+
+For more details about what is happening behind the scenes, read more about
+[portable claims in Crossplane][portable-claims].
+
+## Configure Network Connection
+
+After the Wordpress stack is installed, we will need the AKS Cluster it
+provisions to be able to communicate with the MySQL database it provisions. In
+Azure, we can do so using a [Virtual Network Rule][vnet-rule]. However, the rule
+cannot be created until after the MySQLInstance claim is created and satisfied,
+so we will start a short script to continually check if the database exists, and
+will create the rule if so.
+
+```bash
+cat > vnet-rule.yaml <<EOF
+apiVersion: database.azure.crossplane.io/v1alpha2
+kind: MysqlServerVirtualNetworkRule
+metadata:
+  name: demo-vnet-rule
+  namespace: azure-infra-dev
+spec:
+  name: demo-vnet-rule
+  serverName: MYSQL_NAME
+  resourceGroupName: demo-rg
+  properties:
+    virtualNetworkSubnetId: /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/demo-rg/providers/Microsoft.Network/virtualNetworks/demo-vnet/subnets/demo-subnet
+  providerRef:
+    name: demo-azure
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+EOF
+
+cat <<EOF > vnetwatch.sh
+#!/bin/bash
+
+set -e
+
+echo -n "waiting..."
+while kubectl -n azure-infra-dev get mysqlservers -o yaml | grep -q  'items: \[\]'; do
+  echo -n "."
+  sleep 5
+done
+
+export MYSQL_NAME=$(kubectl -n azure-infra-dev get mysqlservers -o json | jq -j '.items[0].status.providerID')
+
+sed "s/MYSQL_NAME/$MYSQL_NAME/g" vnet-rule.yaml | kubectl create -f -
+
+EOF
+
+chmod +x vnetwatch.sh && ./vnetwatch.sh
+```
+
+## Recap
+
+To recap what we've set up now in our environment:
+
+* Our provider account, both on the provider side and on the Crossplane side.
+* A Resource Group to provision service within.
+* A Network for our AKS Cluster components.
+* A Subnetwork for the AKS cluster to use in the network.
+* A GlobalAddress resource for Google’s service connection.
+* An AKSClusterClass and a SQLServerClass with the right configuration to use
+  the mentioned networking setup.
+* A script that will create our Virtual Network Rule when our MySQL database
+  name comes available.
+* A namespace for our app resources to reside with default MySQLInstanceClass
+  and KubernetesClusterClass that refer to our AKSClusterClass and
+  SQLServerClass.
+
+## Next Steps
+
+Next we'll set up a Crossplane App Stack and use it! Head [back over to the
+Stacks Guide document][stacks-guide-continue] so we can pick up where we left
+off.
+
+## TODO
+This should not go in the final document, but is here for tracking.
+
+* Add references
+* Add next steps, with link to WordPress-specific stuff
+
+<!-- Links -->
```

Comment:
It looks like the line breaks for some of the links got weird; each link should be on its own line.

Reply:
Yep! Accidentally used the line wrap vs code extension on them :/